### DISCUSSION TITLE
Retrieve Moonlight Client Info In Do/Undo Scripts

### DISCUSSION THREAD

**Discussion by jls901** [LizardByte/community#162] [Sunshine] [answered]
Is there a way to get the Moonlight client info (screen resolution and refresh rate) from within a Do/Undo application script?

**Comment by ReenigneArcher** [comment, marked as answer]
See here: https://discord.com/channels/352065098472488960/465368546848014336/1072885292735938590

You need access to Moonlight's discord server and the Android role in order to see that.

**Reply by jls901** [reply]
Yup, that did the trick, thank you!
